tests/release/pipelines/push_to_external_registry.go

```diff
@@ -25,48 +26,62 @@ var _ = framework.ReleasePipelinesSuiteDescribe("Push to external registry", Lab
 
 	var fw *framework.Framework
 	AfterEach(framework.ReportFailure(&fw))
+	var kubeAdminClient *framework.ControllerHub
 	var err error
-	var devNamespace, managedNamespace string
+	var devNamespace = "ex-registry"
+	var managedNamespace = "ex-registry-managed"
 
 	var releaseCR *releaseApi.Release
 	var snapshotPush *appservice.Snapshot
 	var sampleImage = "quay.io/hacbs-release-tests/dcmetromap@sha256:544259be8bcd9e6a2066224b805d854d863064c9b64fa3a87bfcd03f5b0f28e6"
 	var gitSourceURL = releasecommon.GitSourceComponentUrl
 	var gitSourceRevision = "d49914874789147eb2de9bb6a12cd5d150bfff92"
 	var ecPolicyName = "ex-registry-policy-" + util.GenerateRandomString(4)
+	var testEnvironment = utils.GetEnv("TEST_ENVIRONMENT", releasecommon.UpstreamTestEnvironment)
 
 	BeforeAll(func() {
-		// Initialize the tests controllers
-		fw, err = framework.NewFramework(utils.GetGeneratedNamespace("ex-registry"))
-		Expect(err).NotTo(HaveOccurred())
-		devNamespace = fw.UserNamespace
-		managedNamespace = utils.GetGeneratedNamespace("ex-registry-managed")
-		_, err = fw.AsKubeAdmin.CommonController.CreateTestNamespace(managedNamespace)
+		if testEnvironment == releasecommon.DownstreamTestEnvironment {
+			fw, err = framework.NewFramework(utils.GetGeneratedNamespace(devNamespace))
+			Expect(err).NotTo(HaveOccurred())
+			devNamespace = fw.UserNamespace
+			managedNamespace = utils.GetGeneratedNamespace(managedNamespace)
```

Comment:
Below sentence need to be added  in this block?
`kubeAdminClient = fw.AsKubeAdmin`

Reply:
yep, you're right, it should be done similarly as in konflux-demo test [here](https://github.com/konflux-ci/e2e-tests/blob/cc0f13b7d926e4d9f56392e156ba7853758a7e07/tests/konflux-demo/konflux-demo.go#L93-L94)

I will add it in the next commit